test/always.js

```diff
@@ -6,19 +6,26 @@ var R = require('..');
 describe('always', function() {
     it('returns a function that returns the object initially supplied', function() {
         var theMeaning = R.always(42);
-        assert.strictEqual(theMeaning(), 42);
         assert.strictEqual(theMeaning(10), 42);
         assert.strictEqual(theMeaning(false), 42);
     });
 
     it('works with various types', function() {
-        assert.strictEqual(R.always(false)(), false);
-        assert.strictEqual(R.always('abc')(), 'abc');
-        assert.deepEqual(R.always({a: 1, b: 2})(), {a: 1, b: 2});
+        assert.strictEqual(R.always(false)('xxx'), false);
+        assert.strictEqual(R.always('abc')('xxx'), 'abc');
+        assert.deepEqual(R.always({a: 1, b: 2})('xxx'), {a: 1, b: 2});
         var obj = {a: 1, b: 2};
-        assert.strictEqual(R.always(obj)(), obj);
+        assert.strictEqual(R.always(obj)('xxx'), obj);
         var now = new Date(1776, 6, 4);
-        assert.deepEqual(R.always(now)(), new Date(1776, 6, 4));
-        assert.strictEqual(R.always()(), undefined);
+        assert.deepEqual(R.always(now)('xxx'), new Date(1776, 6, 4));
+    });
+
+    it('returns a unary function', function() {
+        assert.strictEqual(R.always(42).length, 1);
+    });
+
+    it('is binary', function() {
+        assert.strictEqual(R.always.length, 2);
+        assert.strictEqual(R.always(42, 'xxx'), 42);
```

Comment:
This seems to be all it takes:

```javascript
/*jshint unused:false */
```

I'll try to check in tonight.  This is working locally for `arity`:

```javascript
module.exports = function(n, fn) { /*jshint unused:false */
    switch (n) {
        case  0: return function() {return fn.apply(this, arguments);};
        case  1: return function(a0) {return fn.apply(this, arguments);};
        case  2: return function(a0, a1) {return fn.apply(this, arguments);};
        case  3: return function(a0, a1, a2) {return fn.apply(this, arguments);};
        case  4: return function(a0, a1, a2, a3) {return fn.apply(this, arguments);};
        case  5: return function(a0, a1, a2, a3, a4) {return fn.apply(this, arguments);};
        case  6: return function(a0, a1, a2, a3, a4, a5) {return fn.apply(this, arguments);};
        case  7: return function(a0, a1, a2, a3, a4, a5, a6) {return fn.apply(this, arguments);};
        case  8: return function(a0, a1, a2, a3, a4, a5, a6, a7) {return fn.apply(this, arguments);};
        case  9: return function(a0, a1, a2, a3, a4, a5, a6, a7, a8) {return fn.apply(this, arguments);};
        case 10: return function(a0, a1, a2, a3, a4, a5, a6, a7, a8, a9) {return fn.apply(this, arguments);};
        default: throw new Error('First argument to arity must be a non-negative integer no greater than ten');
    }
};
```